neqo-bin/src/udp.rs

```diff
@@ -23,6 +23,8 @@ use tokio::io::Interest;
 const RECV_BUF_SIZE: usize = u16::MAX as usize;
 
 pub struct Socket {
+    #[allow(unknown_lints)] // available with Rust v1.75
+    #[allow(clippy::struct_field_names)]
```

Comment:
This one is unrelated. Should I open a separate pull request for it?

Reply:
Let's leave it in.